### ISSUE TITLE
KLE hierarkisk view

### ISSUE THREAD

**Issue by mariushart** [JosefAssad/SeMaWi#103]
Visning af KLE foregår pt. fladt.
KLE er opbygget således at et emne nummer indikerer parten forhold.
Altså Emne AA.BB.CC hvor BB er child til AA og hvor CC er child til BB.

Det ville være ønskeligt at få disse relationer med i visningen.

**Comment by JosefAssad** [comment]
Yes, tilslutter mig ideen. Umiddelbart kan jeg tænke på to steder hvor hierarkiet skal tydeliggøres:

1. På en overordnet KLE side (kan måske anvende [Extension:CategoryTree](https://www.mediawiki.org/wiki/Extension:CategoryTree)). Hvis den virker efter hensigten kan det medføre at vi ændrer lidt i måden vi bruger kategorier i KLE siderne.
2. I forms, så man kan vælge hovedgruppe, derefter Gruppe, derefter Emne (oplagt med [Extension:SemanticFormsSelect](https://www.mediawiki.org/wiki/Extension:Semantic_Forms_Select), men vi afventer en fix i den udvidelse, se [her](https://github.com/SemanticMediaWiki/SemanticFormsSelect/issues/7))

**Comment by JosefAssad** [comment]
Det ligner det bliver mig der skal skrive den patch der fixer SemanticformsSelect. Tror det kan nås i Q2, så måske denne issue i Q3.
